src/inputs/dropdownMenu/dropdownMenu.jsx

```diff
@@ -230,7 +245,7 @@ function DropdownMenu(props) {
         onDropdownMenuReposition();
 
         return () => {
-            if (isOpen) {
+            if (!isOpen) {
```

Comment:
Is this change intentional?  Can I understand why?

Reply:
Yes it's intentional. We want the listener events to be active when the menu is _open_ because we want to be able to close the menu by clicking on the document outside the dropdown. If the events are removed when the menu is open, then the `onClickOutside` will never get run, right?

Reply:
Yeah, you're making sense.  Like we want to remove the events when it is closed.  I wonder was this just a long-running bug?

Reply:
I think so...the deprecated `DropdownMenu` works correctly with similar code, so I think it was just an oversight.